packages/pipeline-editor/src/DialogSelector.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+
+export interface IProps {
+  handleUpdate?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  className: string;
+  id: string;
+  label: string;
+  optionList: JSX.Element[];
+  value?: string;
+}
+
+export const DialogSelector: React.FC<React.HTMLProps<HTMLSelectElement> &
```

Comment:
you could probably do `interface IProps extends React.HTMLProps<HTMLSelectElement> {` above so there's not as much going on here